src/systems/ec/implementations/react/packages/inpage-navigation/examples/Default.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import StoryWrapper from '@ecl/story-wrapper';
+import VanillaInpageNavigation from '@ecl/ec-component-inpage-navigation';
+import demoContent from '@ecl/ec-specs-inpage-navigation/demo/data';
+import InpageNavigation from '../src/InpageNavigation';
+
+const loremIpsum = require('lorem-ipsum');
```

Comment:
why do you switch from `import` to `require`? is there any particular reason?

Reply:
yeah, a blind copy paste from the npm page for this package... :)